src/JuliusSweetland.OptiKey.Core/UI/ViewModels/Management/PointingAndSelectingViewModel.cs

```diff
@@ -129,6 +129,20 @@ public PointingAndSelectingViewModel()
             }
         }
 
+        public List<Tuple<string, int>> GazeSmoothingLevels
+        {
+            get
+            {
+                return new List<Tuple<string, int>>
+                {
+                    new Tuple<string, int>("0 - " + Resources.NONE, 0),
+                    new Tuple<string, int>("1 - " + Resources.LOW, 1),
+                    new Tuple<string, int>("2 - " + Resources.MEDIUM, 25),
+                    new Tuple<string, int>("3 - " + Resources.HIGH, 50),
+                };
+            }
+        }
+
```

Comment:
@JuliusSweetland For the sake of organization and consistency in design, is this how you would supply the GazeSmoothingLevels, or would you create a class, or something else?

Reply:
Yes this is fine. I would use a Tuple of something noddy like this, rather than create another class which is really just a KeyValuePair.

Reply:
Although having said that, just above it is an example of using a KVP: new KeyValuePair<string, DataStreamProcessingLevels>

You could switch to that for a more explicit binding in the XAML - that way you really have Key and Value properties to bind against, instead of Item1, Item2.